common/src/main/proto/TransportMessages.proto

```diff
@@ -433,3 +433,28 @@ message PbResourceConsumption {
   int64 hdfsBytesWritten = 3;
   int64 hdfsFileCount = 4;
 }
+
+message PbAppDiskUsage {
+  string appId = 1;
+  int64 estimatedUsage = 2;
+}
+
+message PbAppDiskUsageSnapShot {
```

Comment:
SnapShot => Snapshot, it's a single word

Reply:
Done